.github/workflows/build.yml

```diff
@@ -75,8 +75,8 @@ jobs:
       - name: Bootstrap executable
         run: ./bin/index.dev.js bundle -p spago-bin
 
-      - name: Check code format
-        run: npm run format:check
```

Comment:
I introduced this check a few PRs ago, but completely missed that this step already exists on line 84.